### PR TITLE
ADAPT-3202 | @mattanglin | campaign header homelink

### PR DESCRIPTION
**Pull request by mattanglin**
# Ready for Review
- (Edit the above to reflect status)

# Summary
Adds visibility toggle for campaign header home link (top right corner)
https://stanfordits.atlassian.net/browse/ADAPT-3202

# Review By (Date)
- August 5th, 2021 @ 4:00pm (PDT) (For sync w/ OOD)

# Criticality
- 1

# Review Tasks

1. Check out this branch
2. Setup env with Storyblok key for  DEV - OOD Giving
3. Enable "Hide Home Link" field for the [test story in dev](https://app.storyblok.com/#!/me/spaces/122401/stories/0/0/65417732/blok/d5ff285c-3243-431a-8e18-1c584c621863?s=1)
4. Run gatsby locally and navigate to http://localhost:8000/campaign-page-2
5. verfiy that home link is hidden
6. Uncheck ("Disable") "Hide Home Link" field
7. Refresh GraphQL cache
8. Verify that home link is visible

### CODE REVIEW

**Review by yvonnetangsu — APPROVED** [review]
Looks good to me - GTG. Thanks!

**Comment by yvonnetangsu** [comment]
@mattanglin What's our new workflow now? Is it feature merge into dev branch, then dev merge into main; or is it feature merge into dev, then feature merge into main? 😄

**Comment by yvonnetangsu** [comment]
Merging 😄

**Comment by mattanglin** [comment]
> @mattanglin What's our new workflow now? Is it feature merge into dev branch, then dev merge into main; or is it feature merge into dev, then feature merge into main? 😄

@yvonnetangsu Sorry, just saw this now.

**tl;dr** feature into dev, cut release branch from dev into

This repo is in a kind of "in between" state presently, but we've been talking about moving toward what is essentially a [gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) workflow. Gitflow uses `dev` branch as an integration branch (which deploys to a pre-production environment) for all feature work and then release branches are created from `dev` and promoted to `main` (which deploys to production).
The only other time PRs are made against `main` is for hotfixes when there is work in `dev` that we don't want to release to production. Hopefully that will generally be a non-issue as I'd prefer more continuous delivery anyhow, but that will mostly be dependent on the nature of the project (if it's already live or not) and the whim of the client.

**Comment by yvonnetangsu** [comment]
> > @mattanglin What's our new workflow now? Is it feature merge into dev branch, then dev merge into main; or is it feature merge into dev, then feature merge into main? 😄
> 
> @yvonnetangsu Sorry, just saw this now.
> 
> **tl;dr** `feature` into `dev`, cut `release` branch from `dev` into `main`
> 
> This repo is in a kind of "in between" state presently, but we've been talking about moving toward what is essentially a [gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) workflow. Gitflow uses `dev` branch as an integration branch (which deploys to a pre-production environment) for all feature work and then release branches are created from `dev` and promoted to `main` (which deploys to production).
> The only other time PRs are made against `main` is for hotfixes when there is work in `dev` that we don't want to release to production. Hopefully that will generally be a non-issue as I'd prefer more continuous delivery anyhow, but that will mostly be dependent on the nature of the project (if it's already live or not) and the whim of the client.

Thanks, @mattanglin That sounds great to me!
